src/content-script/index.mjs

```diff
@@ -1,38 +1,67 @@
-import Browser from "webextension-polyfill";
+import Browser from 'webextension-polyfill'
+import {getPossibleElementByClassArray, getPossibleElementByIdArray, getPossibleElementByNameArray} from './utils.mjs'
+
+const config = {
```

Comment:
Instead of doing it this way which will get messy and harder to maintain. Having a configuration for each search engine then it will be a single lookup instead doing it for all arrays. Easier to maintain if we split this up per search engine.

Reply:
agree with you, this is a 'just work' version

Reply:
Much better!